lib/database/showtimes.py

```diff
@@ -2,7 +2,7 @@
 import rethinkdb as r
 from dateutil import tz
 
-from . import encryption
+from .encryption import *
```

Comment:
this'll give an error on line 46.  Try changing to `from .encryption import create_showtime_keys` and then change line 46 to `yield create_showtime_keys(show_id)`?